tests/behavioural/features/steps/body_field.py

```diff
@@ -31,8 +31,29 @@ def step_impl_the_msg_body_is_set_too_long(context):
     context.bdd_helper.message_data['body'] = "x" * (constants.MAX_BODY_LEN + 1)
 
 
+@given("the message body is '{body_length}' characters long")
```

Comment:
This is a duplicate function name , the test will still pass but it will cause confusion later . A separate function name is preferable . Found this by running pylint